### PR TITLE
Jeremy likness minapi overview

### LINKED ISSUE #27600: Create Overview of Minimal APIs
~@rafikiassumani-msft~ @JeremyLikness   just get a solid draft and I'll polish it off.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: b09824ab-b051-75a1-13b7-b0e281cc46e0
* Version Independent ID: b09824ab-b051-75a1-13b7-b0e281cc46e0
* Content: [Minimal APIs quick reference](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis?view=aspnetcore-7.0)
* Content Source: [aspnetcore/fundamentals/minimal-apis.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/fundamentals/minimal-apis.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-fundamentals**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by JeremyLikness**
Fixes #27600 

Draft of minimal API overview.

***Edit by @Rick-Anderson :***
[Internal review URL](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis/overview?view=aspnetcore-7.0&branch=pr-en-us-27724)

### CODE REVIEW

File: aspnetcore/fundamentals/minimal-apis/overview.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+description: An introduction to the fastest and easiest way to create web API endpoints with ASP.NET Core.
+ms.author: jeliknes
+monikerRange: '>= aspnetcore-6.0'
+ms.date: 11/22/2022
+uid: fundamentals/minimal-apis/overview
+---
+
+# Minimal APIs overview
+
+Minimal APIs are both a technology and an approach to quickly and easily create web-based APIs with HTTP endpoints in ASP.NET Core.
```
Comment (rafikiassumani-msft) [inline aspnetcore/fundamentals/minimal-apis/overview.md:13]: Can we rephrase this to : "Minimal APIs are a ASPNET Core micro-framework for quickly and easily creating HTTP APIs with .NET" 

or "Minimal APIs are a simplified approach for building fast HTTP APIs with ASPNET Core". 

A few things:
- Can we avoid "Web-based"? 
- Can we make sure the "Minimalistic approach" and "perf=fast" show up in the wording ?

File: aspnetcore/fundamentals/minimal-apis/overview.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+var app = builder.Build();
+
+app.MapGet("/users/{userId}/books/{bookId}", 
+    (int userId, int bookId) => $"The user id is {userId} and book id is {bookId}");
+
+app.Run();
+```
+
+That's all it takes to get started, but it's not all that's available. Minimal APIs support the configuration and customization you need to scale to multiple APIs, handle complex routes, apply authorization rules and control the shape and content of your API responses. To learn more, use the table of contents navigation to jump to the topic of your choice. A good place to get started is <xref:fundamentals/minimal-apis/webapplication>.
```
Comment (tdykstra) [inline aspnetcore/fundamentals/minimal-apis/overview.md:38]: I would move the [Differences between minimal APIs and APIs with controllers](https://learn.microsoft.com/en-us/aspnet/core/tutorials/min-web-api?view=aspnetcore-7.0&tabs=visual-studio#differences-between-minimal-apis-and-apis-with-controllers) section from the tutorial to here.

Consider offering the tutorial as "A good place to get started" rather than the cheat sheet.

```suggestion
That's all it takes to get started, but it's not all that's available. Minimal APIs support the configuration and customization you need to scale to multiple APIs, handle complex routes, apply authorization rules, and control the content of your API responses. A good place to get started is <xref:tutorials/min-web-api>.
```

File: aspnetcore/fundamentals/minimal-apis/overview.md (hunk 1 above)
Comment (Rick-Anderson) [reply aspnetcore/fundamentals/minimal-apis/overview.md:13]: > Can we rephrase this to : "Minimal APIs are a ASPNET Core micro-framework for quickly and easily creating HTTP APIs with .NET"

@JeremyLikness  I'll suggest a change with this and other changes.

**Review by tdykstra — COMMENTED** [review]
Looks good, a couple of suggestions.

File: aspnetcore/fundamentals/minimal-apis/overview.md (hunk 2 above)
Comment (Rick-Anderson) [reply aspnetcore/fundamentals/minimal-apis/overview.md:38]: slight tweak to @tdykstra 
shorter is always better per MS style guide

```suggestion
That's all it takes to get started, but it's not all that's available. Minimal APIs support the configuration and customization needed to scale to multiple APIs, handle complex routes, apply authorization rules, and control the content of API responses. A good place to get started is <xref:tutorials/min-web-api>.
```

**Comment by Rick-Anderson** [comment]
@JeremyLikness  ready for your review

**Review by JeremyLikness — COMMENTED** [review]
Looks great, thanks.
